internal/webhooks/tempostack_webhook.go

```diff
@@ -149,6 +149,20 @@ func (d *Defaulter) Default(ctx context.Context, obj runtime.Object) error {
 		r.Spec.Template.QueryFrontend.JaegerQuery.ServicesQueryDuration = &defaultServicesDuration
 	}
 
+	if d.ctrlConfig.Gates.OpenShift.OauthProxy.DefaultEnabled {
+		if r.Spec.Template.QueryFrontend.JaegerQuery.Enabled && r.Spec.Template.QueryFrontend.JaegerQuery.Ingress.Type == v1alpha1.IngressTypeRoute {
+			r.Spec.Template.QueryFrontend.JaegerQuery.Oauth.Enabled = true
```

Comment:
If the user explicitly sets `r.Spec.Template.QueryFrontend.JaegerQuery.Oauth.Enabled` to false, the current code overwrites it every time the webhook runs (if `DefaultEnabled` = true).

So it's not a default anymore, but more a `ForceEnabled` :smile:

Possible solutions
* make the bool a pointer, then we can differentiate between unset, true and false
* remove this setting from the CR, i.e. don't make it user-configurable per tempo instance

Reply:
True! I'll fix this.

Reply:
Fixed!